lib/cdo/aws/cloudfront.rb

```diff
@@ -135,6 +135,19 @@ def self.distribution_config(app, origin, aliases, ssl_cert=nil)
       # Add root-domain aliases to production environment stack.
       aliases += CONFIG[app][:aliases] if rack_env?(:production)
 
+      origin_custom_headers = [
+        {
+          HeaderName: 'X-Cdo-INF506',
+          HeaderValue: '17e071a4eeb0'
```

Comment:
I think we can declare a configuration setting in `/config.yml.erb` that is populated from an AWS Secret. Something like
`auth_forwarded_by_cloudfront: !Secret`

And then populate that in the template with something like:
`HeaderValue: '{{resolve:secretsmanager:<%=environment%>/cdo/auth_forwarded_by_cloudfront}}'`

Reply:
We would need to generate the secret values for each environment locally and then publish them to Secrets Manager from a dev environment [via these steps](https://github.com/code-dot-org/code-dot-org/blob/staging-next/config/secrets.md).